pyspecdata/core.py

```diff
@@ -2500,6 +2555,15 @@ def __mul__(self, arg):
             Rerr = None
         # }}}
         retval.set_error(Rerr)
+        unitA = Q_(
+            "dimensionless" if self.get_units() is None else self.get_units()
+        )
+        unitB = Q_(
+            "dimensionless" if arg.get_units() is None else arg.get_units()
+        )
+        unit_ret = f"{(unitA*unitB).units:~P}"
+        unit_ret = None if len(unit_ret) == 0 else unit_ret
+        retval.set_units(unit_ret)
```

Comment:
to handle unit conversion if the error is in different units from the data or axis